util/design/lib/OtpMemMap.py

```diff
@@ -548,6 +548,7 @@ def create_partitions_table(cls, partitions: ConfigT) -> str:
             is_buffered = "yes" if part["variant"] in [
                 "Buffered", "LifeCycle"
             ] else "no"
+            is_zeroizable = "yes" if check_bool(part["zeroizable"]) else "no"
```

Comment:
A future PR could clean up the repetitions of these kinds of lines and factor that to a common `bool_to_yes_no` function